testing/tests/DevExpress.exporter/jspdfParts/jspdfTestHelper.js

```diff
@@ -6,6 +6,24 @@ class JSPdfDataGridTestHelper {
         this.jsPDFDocument = jsPDFDocument;
     }
 
+    checkCustomizeCell(eventArgs, expectedCells, callIndex) {
+        assert.ok(eventArgs.gridCell);
+        assert.ok(eventArgs.pdfCell);
+
+        const { head = [], body = [] } = expectedCells;
+        const expectedPdfCells = head.concat(body);
+
+        const actualPdfCell = eventArgs.pdfCell;
+        let index = 0;
+        this._iterateCells(expectedPdfCells, (expectedPdfCell, rowIndex, columnIndex) => {
+            if(index === callIndex) {
```

Comment:
Do this condition is required?

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/15039/commits/961ded0817b3666c0a70b4d9cb2f065cfecaac0a

Reply:
fixed